charts/sn-platform/templates/tls/istio/tls-certs.yaml

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+{{- if .Values.istio.enabled }}
+{{- if or .Values.certs.istio_internal_issuer.enabled .Values.certs.istio_public_issuer.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: "{{ template "istio.credentialName" . }}"
+  namespace: {{ template "istio.namespace" . }}
+spec:
+  secretName: "{{ template "istio.credentialName" . }}"
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  isCA: false
+  privateKey:
+    algorithm: RSA
+    encoding: PKCS1
+    size: 2048
+  usages:
+    - server auth
+    - client auth
+  dnsNames:
+  - "*.{{ .Values.domain.suffix }}"
```

Comment:
It covers only one case, while the user may also define `ingress.[control_center/broker].external_domain` as well.

Reply:
Ref the previous public cert definition: https://github.com/streamnative/charts/blob/0bba32fc686a561e4280d36ed9ccaf698ca37fc1/charts/sn-platform/templates/tls/tls-certs-public.yaml#L32-L36
If `ingress.[control_center/broker].external_domain` is covered by `*.{{ .Values.domain.suffix }}`, Certificate will be invalid.